### PR TITLE
xplanet 1.3.1

### PR DESCRIPTION
**Pull request by blogabe**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Does your submission pass `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

File: Formula/xplanet.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Xplanet < Formula
   desc "Create HQ wallpapers of planet Earth"
   homepage "http://xplanet.sourceforge.net/"
-  url "https://downloads.sourceforge.net/project/xplanet/xplanet/1.3.0/xplanet-1.3.0.tar.gz"
-  sha256 "44fb742bb93e5661ea8b11ccabcc12896693e051f3dd5083c9227224c416b442"
+  url "https://downloads.sourceforge.net/project/xplanet/xplanet/1.3.1/xplanet-1.3.1.tar.gz"
+  sha256 "4380d570a8bf27b81fb629c97a636c1673407f4ac4989ce931720078a90aece7"
   revision 2
```
Comment (UniqMartin) [inline Formula/xplanet.rb]: Please remove this revision line when bumping the version of the formula.

Comment (UniqMartin) [inline Formula/xplanet.rb]: (This should have been pointed out by the `brew audit --strict --online <formula>`.)

**Comment by UniqMartin** [comment]
> - [x] Have you built your formula locally prior to submission with `brew install <formula>`?

Are you sure you did this? I would be a bit surprised if it builds on your machine but fails on our test bot on all three OS X versions we're testing. One reason for the CI failure is that some of the patches no longer apply. Can you please check if they have been merged upstream and can be omitted or if they need to be adapted for the new version? Thanks!

**Comment by blogabe** [comment]
Thanks for clarifying.  Giflib 4.2.3 (and latest version of 5) released in July '15 rolls in the patch update which were included in this formula in June '15.  Should be good to go.

**Comment by UniqMartin** [comment]
Thanks, the build succeeds now!

The formula unfortunately doesn't have a test yet, which makes it harder for our build infrastructure to check that it's not affected by other updates and makes it more likely to break in the future. Would you be willing to try and add a test to the formula? [This section of the Formula Cookbook](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Formula-Cookbook.md#add-a-test-to-the-formula) should get you started, and I'm happy to answer any related questions you might have. It's okay if you rather not, but it would be much appreciated if you could give this a try.

**Comment by blogabe** [comment]
Just trying to update the formula to the latest version, but I did add the test block for you.  I tested it on my system and it works.  If it doesn't meet the necessary requirements, then I'll rely on you to fix/complete the test block.  Sound good?

**Comment by UniqMartin** [comment]
> Just trying to update the formula to the latest version, but I did add the test block for you.

I know, we all want just the latest software. :smiley:

But you took the time to create a test and for that we're really grateful! It looks great and makes me feel optimistic that it will also work on our test bot. I'll merge it as soon as it's ready.

**Comment by blogabe** [comment]
Great.  Thank you!

**Comment by UniqMartin** [comment]
Merged in 7503a180f78be6c9a5cc7517f6bbe705d8136b5a. Thank you for this contribution to Homebrew, @blogabe! You rock! :tada:

**Pull request closed without merging**